testing/endtoend/components/eth1/transactions.go

```diff
@@ -110,11 +110,11 @@ func SendTransaction(client *rpc.Client, key *ecdsa.PrivateKey, f *filler.Filler
 		g.Go(func() error {
 			tx, err := txfuzz.RandomValidTx(client, f, sender, nonce+index, gasPrice, nil, al)
 			if err != nil {
-				return nil
+				return err
 			}
 			signedTx, err := types.SignTx(tx, types.NewLondonSigner(chainid), key)
 			if err != nil {
-				return nil
+				return err
 			}
 			err = backend.SendTransaction(context.Background(), signedTx)
 			return nil
```

Comment:
```suggestion
			return backend.SendTransaction(context.Background(), signedTx)
```

Another nil err return here